hw/ip/i2c/doc/_index.md

```diff
@@ -142,6 +148,34 @@ Typically every byte transmitted must also receive an ACK signal, and the IP wil
 However, there are some I2C commands which do not require an ACK.
 In those cases this flag should be asserted with FBYTE indicating no ACK is expected and no interrupt should be raised if the ACK is not received. 
 
+### Target Address Registers
+
+I2C target device is assigned two 7-bit address and 7-bit mask pairs.
+The target device accepts a transaction if the result of the bitwise AND operation performed on the transaction address sent by the host and a mask matches the assigned address corresponding to the mask.
+Thus, with the masks set to all ones (0x7F), the target device will respond to either of the two assigned unique addresses and no other.
+If the mask and the assigned address both have zeros in a particular bit position, that bit will be a match regardless of the value of that bit received from the host.
+Note that if, in any bit position, the mask has zero and the assigned address has one, no transaction can match and such mask/address pair is effectively diasabled.
```

Comment:
This all seems unnecessarily complex.  Couldn't you just simplify this to say that "address match-checking is only performed for bits where the mask is '1'"?  I know that is slightly different than what you have said here, but as an alternative functionality it would be easier to understand. (and simple enough to check i.e. `(RECD_ADDR & MASK) == (REG_ADDR & MASK)`)

Reply:
Am I missing something about the intent?  If so, could you please elaborate?

Reply:
I have inserted such a statement close to the beginning.